src/main/java/com/renatusnetwork/momentum/Momentum.java

```diff
@@ -227,6 +230,7 @@ public void run()
         bank = new BankManager();
         blackmarket = new BlackMarketManager();
         cmdSigns = new CommandSignManager();
+        squadManager = new SquadManager();
```

Comment:
This also means that SquadManager should be SquadsManager and other instances of getSquadManager to SquadsManager etc.